articles/web-dev's-guide-SEO/index.md

```diff
@@ -0,0 +1,85 @@
+
+## PageRank
+The [PageRank](https://en.wikipedia.org/wiki/PageRank) algorithm, named after Larry Page, is the algorithm used to measure and rank the importance of websites. The underlying assumption that ranking algorithms make is as follows: the quality of a site is directly proportional to the number of links to it. Quality links are links to pages that are ranked better. 
+
+Now that we have sufficient background about the topic, let us look at some pro-SEO dev techniques to boost the rankings higher and higher.
+
+### Metadata
+
+1. **Adding meta tag**: Adding meta tags is the oldest way of implementing SEO. These meta tags are placed inside the head tag along with other link tags. Let us look at examples for the same.
+   ```html    
+   <meta name="description" content="Welcome to the blog on SEO on Section.io "/>    
+   <meta name="keywords" content="SEO, Web dev, pagerank, node.js "/>
+    ```
+    The description and keywords are two such options that crawlers look for. Keywords are not used much today, but it's a good practice to include them. 
+
+2. **Adding title tag**: The title tag displays the current title of the webpage. A common mistake that developers make is having the same title tag for most of the pages on a website. The ranking algorithms penalize such behavior of websites for lack of information. The title tag must be modified according to the content at hand. For example, look at the current tab. Is the title " Developer's Guide to Search Engine Optimization| Section"? Such optimization is required for getting better results at search engines.   
+
+3. **Adding favicons:** Favicons are the tiny icons that appear on the tabs. They don't directly affect SEO. They play a vital role in optimizing the user experience. The favicons represent the brand and therefore have huge importance. Looking at the f icon on top of the Facebook page helps us easily identify the tab amidst other tabs. The favicon also helps to identify the webpage in bookmark lists as well as search history. Better and easier identification of websites enables larger time spent on the site thereby optimizing it.
+
+### Social Metadata
+As discussed earlier, controlling the experience of the user upon the clicking of an URL is very important. Various technologies and protocols are built to enhance the same. 
+
+#### Facebook Open Group Protocol
+
+Facebook's [Open Graph Protocol](https://developers.facebook.com/docs/sharing/opengraph/) decides how URLs are shown in Facebook posts. There are various meta tags that one can add. It is the same as before, except a property tag is added.
+
+```html
+<meta property="og:proprty_name" content="canonicalURL">
+```
+og stands for Open Graph. The ```property_name``` mentioned in meta tag after og takes several values. They are as follows:
+1. title
+2. type
+3. URL
+4. image
+5. admins
+6. site_name
+7. description 
+
+Let us look at how these property names help enhance the user experience in a Facebook post.
```

Comment:
You could reference Facebook's tool for validating OpenGraph meta tags. https://developers.facebook.com/tools/debug/